ssc/cmod_pvwattsv8.cpp

```diff
@@ -826,12 +823,28 @@ class cm_pvwattsv8 : public compute_module
                 else if (is_assigned("albedo"))
                     log(util::format("Albedo array was assigned but is not the correct length (1, 12, or %d entries). Using default value.", nrec), SSC_WARNING);
 
-                // if the user hasn't specified an albedo, or has specified to use wf albedo, and the weather file contains hourly albedo, use that instead
-                // but make sure that the weather file albedo is realistic
-                // albedo_len will be zero if the albedo input isn't assigned
-                if (std::isfinite(wf.alb) && wf.alb > 0 && wf.alb < 1 && (albedo_len == 0 || use_wf_albedo))
+                // check albedo is valid value between 0 and 1. if not, set to default value 0.2
+                // if the snow loss model is enabled and there's valid snow depth > 0.5 cm, then set to 0.6
+                if (alb <= 0 || alb >= 1)
```

Comment:
If albedo_len is wrong, pvwatts will never use the snow value because the albedo default set in line 816 won't trigger this if statement. Suggest replacing line 816 with the contents of this if statement.

Reply:
Fixed in 22acecb6